data/actionsmap/yunohost.yml

```diff
@@ -302,6 +302,14 @@ domain:
             arguments:
                 domain:
                     help: Target domain
+                -ttl:
```

Comment:
Word arguments might have double dash, and I think we should provide one letter argument like
```
                 -t:
                     full: --ttl
```